### ISSUE TITLE
NPC's minions don't turn on combat.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#855]
Something really off happened earlier to me.
Was questing in Wetlands and i bodypulled a minion (only the minion!!) called Skeleton (Dragonmaw Bonewarder's Minion) and it was running towards me, targetting me, ATTACKING ME, and it didn;t put me in combat except for the split second of me receiving the actual damage.
https://i.imgur.com/P1L4D1e.jpg

**Comment by ifyousayso** [comment]
Yeah, these minions, along with Westfall Crawlers, never stop chasing (and aren't affected by threat in any way).

Murloc Hunter: http://db.vanillagaming.org/?npc=458
Crawler: http://db.vanillagaming.org/?npc=6250
